src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Completion.Providers;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Recommendations;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    [ExportCompletionProvider(nameof(UnnamedSymbolCompletionProvider), LanguageNames.CSharp), Shared]
+    [ExtensionOrder(After = nameof(SymbolCompletionProvider))]
+    internal partial class UnnamedSymbolCompletionProvider : LSPCompletionProvider
+    {
+        /// <summary>
+        /// CompletionItems for indexers/operators should be sorted below other suggestions like methods or properties
+        /// of the type.  We accomplish this by placing a character known to be greater than all other normal identifier
+        /// characters as the start of our item's name. this doesn't affect what we insert though as all derived
+        /// providers have specialized logic for what they need to do.
+        /// </summary> 
+        private const string SortingPrefix = "\uFFFD";
```

Comment:
Not blocking this PR, but I think we now have a couple providers using trick like this to control where and how a group of related item to be displayed in the list. We should consider centralize the definition/creation of those for maintainability.
e.g. 
http://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/Completion/CompletionProviders/DeclarationNameCompletionProvider.cs,80
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/Providers/AbstractPreprocessorCompletionProvider.cs,54

Reply:
Yes. I agree. Let me see if I can figure something out here.

Reply:
for now i'm going to leave as is.  what i think would make the most sense would be to allow providers for a particular service to have before/after ordering.  And we can generate an appropriate sort-text prefix for each that respects this.  but i don't really want to do this now :)